clients/lsp-rust.el

```diff
@@ -585,6 +585,11 @@ The command should include `--message=format=json` or similar option."
 (lsp-defun lsp-rust--analyzer-run-single ((&Command :arguments?))
   (lsp-rust-analyzer-run (lsp-seq-first arguments?)))
 
+(lsp-defun lsp-rust--analyzer-show-references
+  ((&Command :title :arguments? [_uri _filepos references]))
+  (lsp-show-xrefs (lsp--locations-to-xref-items references) nil
+                  (s-contains-p "reference" title)))
```

Comment:
out of curiosity what are the expected `title` values here?

Reply:
<any number> reference(s)
<any number> definition(s)
E.g. 2 references, 2 definitions. I'm unsure whether reference can actually be encountered in the wild.

Reply:
As a side note, do we prefer scheme-style `?` or common-lisp-style `p` predicate functions? I.e. `s-contains-p` or `s-contains?`.

Reply:
> As a side note, do we prefer scheme-style `?` or common-lisp-style `p` predicate functions? I.e. `s-contains-p` or `s-contains?`.

Most of the code is written using scheme-style although this is not enforced 100% during reviews.